src/pybind/mgr/volumes/fs/subvolume.py

```diff
@@ -143,6 +143,38 @@ def remove_subvolume(self, spec, force):
         except cephfs.Error as e:
             raise VolumeException(-e.args[0], e.args[1])
 
+    def resize_subvolume(self, subvolpath, newsize, limitshrink):
+        """
+        :param subvolpath: subvolume path
+        :param newsize: new size In bytes
+        :return: new quota size and used bytes as a tuple
+        """
+        if newsize <= 0:
+            raise VolumeException(-errno.EINVAL, "Provide a valid size")
+
+        try:
+            maxbytes = int(self.fs.getxattr(subvolpath, 'ceph.quota.max_bytes').decode('utf-8'))
+        except cephfs.NoData:
+            pass
+
+        usedbytes = int(self.fs.getxattr(subvolpath, "ceph.dir.rbytes").decode('utf-8'))
+
+        if newsize == maxbytes:
+            log.warn("The resize value '{0}' is equal to the current quota size '{1}'".format(newsize, usedbytes))
```

Comment:
Because this interface will be used programatically and not by admins, this could result in a lot of spam to the mgr log. Let's just get rid of this warning entirely.